### PR TITLE
[RDY] Redo annotations

### PR DESCRIPTION
**Pull request by pvanagtmaal**
Draw them differently
Remove redundant info
Color consistently

### CODE REVIEW

File: dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/annotations/Gene.java [hunk 1]
```diff
@@ -21,13 +22,13 @@ public Gene(Annotation annotation) {
...
-		getChildren().addAll(name, coordinates, sense);
-
-		// FIXME: Jente: probably replace this in redo-annotations?
+		Tooltip tooltip = new Tooltip("Coordinates: " + annotation.getStart() + " to "
+				+ annotation.getEnd() + "\n" + "Sense: " + annotation.isSense());
+		Tooltip.install(name, tooltip);
+		getChildren().add(name);
 		if (annotation.isMutation()) {
-			setStyle("-fx-background-color: #FF0000");
+			// FIXME: Jente: probably replace this in redo-annotations?
```
Comment (Unia) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/annotations/Gene.java]: This should be a style class defined in css.

**Comment by Unia** [comment]
LGTM!

File: dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java [hunk 2]
```diff
@@ -116,39 +118,18 @@ private boolean needRedraw(Range ranks, Thresholds newThreshold) {
...
 					Bindings.divide(right.translateXProperty().subtract(left.translateXProperty()),
 							2), g.widthProperty().divide(2))));
-			if (prev != null && g.getLayoutBounds().intersects(prev.getLayoutBounds())) {
-				g.translateYProperty().bind(prev.translateYProperty()
-						.add(prev.heightProperty()).add(ANNOTATION_HEIGHT));
-			} else {
-				g.translateYProperty().bind(left.translateYProperty().add(ANNOTATION_HEIGHT));
-			}
+			g.translateYProperty().setValue(ANNOTATION_HEIGHT
+					* (curAnnotationY++ % MAX_ANNOTATIONS));
```
Comment (Balletie) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java]: At least it's not an infinite sequence..

**Comment by Balletie** [comment]
Waiting for travis.

**Comment by gfokkema** [comment]
Works nice.

**Pull request merged**